### DISCUSSION TITLE
What is `10-page markers` in Book map

### DISCUSSION THREAD

**Discussion by Monirzadeh** [koreader/koreader#11786] [Q&A] [answered]
Hi.
if you open book map and tap on hamburger menu you will see an option name `10-page markers` that you can change that with `-` and `+`.
i can't see any change when i increase or decrease that. anybody know what is it and what exactly do?

**Comment by poire-z** [comment, marked as answer]
It's a little indicator every 10 pages, below the rectangles - the -/+ just sets their height and how obvious they are (and some additional smaller one every 5 pages if you choose the larger+).
Some screenshots where you will notice may be better than on your device in https://github.com/koreader/koreader/issues/10433#issuecomment-1547793914.
Been discussed all along that github issue (and it's optional and tweakable with this -/+ because I don't like "hairy boxes" :))

**Reply by Monirzadeh** [reply]
Thanks for explain :heart:

**Reply by jonnyl2** [reply]
>I don't like "hairy boxes" :)

Yet most people seem to love them :). Most Book map screenshots I've seen have had them enabled :)
